### ISSUE TITLE
Setting controlOptions doesn't work as expected

### ISSUE THREAD

**Issue by mhp-borisbojic** [hujiulong/vue-3d-model#126]
From the code and documentation, I understand that the prop "controlsOption" expects an object with parameters from the OrbitControls, for example:

**EDIT: the problem is your npm package - it's not up-to-date, yet. Could you provide a latest one? :)**

```
<template>
  <div id="app">
    <model-gltf
      src="/assets/objects/Wing2_optimized.gltf"
      :controlsOptions="controlsOptions"
    />
  </div>
</template>

<script>
import { ModelGltf } from 'vue-3d-model';
export default {
  name: 'App',

  components: {
    ModelGltf
  },

  data: function() {
    return {
      controlsOptions: {
        enablePan: false,
        enableZoom: false
      }
    }
  },

}
</script>

<style lang="scss">
#app {
  margin: 0 auto;
  width: 80%;
  border: 1px solid red;
}
</style>
```

_I want to disable any zoom, but even with these settings, the zoom still works (with mouse and touchpad)._

_What am I doing wrong?_

**Comment by hujiulong** [comment]
This feature is already included in v1.2.0. Try to install this version

**Comment by mhp-borisbojic** [comment]
@hujiulong 

Sadly, now your npm package doesn't work anymore at all:

```
This dependency was not found:

* vue-3d-model in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/App.vue?vue&type=script&lang=js&

To install it, you can run: npm install --save vue-3d-model
```

I even tried to use it in a completely fresh vue project - the error message remains.

Reverting to v1.1.1 doesn't help as well. :(
